molecule/scram-rhel/verify.yml

```diff
@@ -5,6 +5,9 @@
   hosts: kafka_broker
   gather_facts: false
   tasks:
+    - set_fact:
+        kraft_mode: "{{ ('kafka_controller' in groups.keys() and groups['kafka_controller'] | length > 0) }}"
```

Comment:
this scenario is not valid in kraft, why is this check needed?

Reply:
Yes true, but it doesnt do any harm to keep it. The reason being if in near future we add the support for scram in kraft then at the time we wont have to make that change in this verify file